scripts/easy-secrets-gen.sh

```diff
@@ -37,7 +39,7 @@ for ns in nautobot dex; do
     -o yaml \
     --type Opaque \
     --from-literal=client-secret="$NAUTOBOT_SSO_SECRET" \
-    > secret-nautobot-sso-$ns.yaml
+    > "${DEST_DIR}/secret-nautobot-sso-$ns.yaml"
```

Comment:
this is really for the section below but since it wasn't changed, I cannot make a comment on it directly:

```
    > secret-argo-sso-$ns.yaml
```

must be changed to:

```
    > ${DEST_DIR}/secret-argo-sso-$ns.yaml
```

Reply:
Yeah I think that was pulled in with PR 28. Fixed now.